modules/button/css/lib/button.scss

```diff
@@ -49,32 +60,58 @@ $_wdc-btn-caret-down-icon-white: svg-load(
   &.wdc-btn-hover,
   &:hover {
     background-color: $wdc-btn-bg-secondary-hover;
+    @include wdc-icon-color(#333d47);
   }
 
   &.wdc-btn-active,
   &:active {
     background-color: $wdc-btn-bg-secondary-active;
+    @include wdc-icon-color(#333d47);
   }
 
   &:focus,
   &.wdc-btn-focus {
     background-color: $wdc-btn-bg-secondary-focus;
     box-shadow: $wdc-btn-focus-ring;
     outline: none;
+    @include wdc-icon-color(#7b858f);
+  }
+
+  &:focus:hover {
+    @include wdc-icon-color(#333d47);
   }
 
   &.wdc-btn-disabled,
   &:disabled {
     background-color: $wdc-btn-bg-secondary-disabled;
     color: $wdc-btn-secondary-disabled-text-color;
     cursor: default;
+    @include wdc-icon-color($wdc-btn-secondary-disabled-text-color);
   }
 
   &.wdc-btn-grow {
     width: 100%;
     max-width: 100%;
   }
 
+  .wdc-icon[data-category="system"]:not([data-circle-background]) {
+    vertical-align: -7px;
+
+    &,
+    .wdc-icon-position-left & {
+      margin: 0 $wdc-spacing-xxs;
+      margin-left: 0;
+    }
+  
+    .wdc-icon-position-right & {
+      margin: 0 $wdc-spacing-xxs;
```

Comment:
Nit: Can we either use `margin-right: $wdc-spacing-xxs;` or `margin: 0 0 0 $wdc-spacing-xxs;`? Same goes elsewhere.